.github/workflows/05-pr-comment.yml

```diff
@@ -5,16 +5,32 @@ on:
     types: [created]
 
 jobs:
+  awaiting_author:
+    if: github.event.issue.pull_request != null && contains(github.event.comment.body, 'awaiting-author')
+    runs-on: ubuntu-latest
+    steps:
+    - name: Add 'awaiting-author; and remove any 'awaiting-review'
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          const { owner, repo, number: issue_number } = context.issue;
+          await github.rest.issues.addLabels({ owner, repo, issue_number, labels: ['awaiting-author'] });
+          await github.rest.issues.removeLabel({ owner, repo, issue_number, name: 'awaiting-review' }).catch(() => {});
+
   awaiting_review:
     if: github.event.issue.pull_request != null && contains(github.event.comment.body, 'awaiting-review')
     runs-on: ubuntu-latest
 
     steps:
-    - name: Add "awaiting-review" label to the PR
```

Comment:
Is there a reason you're switching from curl to javascript here? Consistency with batteries?
(There is also a curl incantation to remove labels, so you could also do this using curl.)

Reply:
Consistency, with the example in batteries.but also imo it's much nicer to read

Reply:
FWIW, mathlib uses/used the curl incantation. Anyway: my personal opinion preferfs curl, but this is really subjective --- so this is no reason for this to not land!